packages/react-dom/src/events/DOMModernPluginEventSystem.js

```diff
@@ -362,11 +399,19 @@ export function listenToReactPropEvent(
   // this React prop event again.
   listenerMap.set(reactPropEvent, null);
   const dependencies = registrationNameDependencies[reactPropEvent];
+  const dependenciesLength = dependencies.length;
+  // If the dependencies length is 1, that means we're not using a polyfill
+  // plugin like ChangeEventPlugin, BeforeInputPlugin, EnterLeavePlugin and
+  // SelectEventPlugin. SimpleEventPlugin always only has a single dependency.
+  // Given this, we know that we never need to apply capture phase event
+  // listeners to anything other than the SimpleEventPlugin.
+  const registrationCapturePhase =
```

Comment:
Nit: can we rename this to better signal what it represents? Like `isSomething`. I wouldn't nitpick but I genuinely struggle to piece together what exactly this Boolean is telling us